stubs/jsonschema/jsonschema/validators.pyi

```diff
@@ -4,18 +4,21 @@ from typing import Any, ClassVar
 
 from ._utils import URIDict
 
+_Schema = Any
```

Comment:
This can apparently be a boolean or `Mapping[str, str]`, or at least it is expected to have a `.get()` method similar to mappings.

Reply:
I didn't have time to investigate this in depth and didn't want to make assumptions, but if you have determined the type I can include it.